Screens/NewRace.js

```diff
@@ -0,0 +1,170 @@
+import React, { Component } from 'react'
+import {
+  Alert,
+  Button,
+  Picker,
+  StyleSheet,
+  Text,
+  TextInput,
+  View
+} from 'react-native'
+
+class NewRace extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      codex: '',
+      gender: 'Female',
+      temperature: '',
+      precipitation: 'None',
+      course: '',
+      racers: 6,
+      position: 'Start Gate',
+    }
+  }
+
+  static navigationOptions = {
+    title: 'New Race'
+  }
+
+  _handleBtnPress() {
+    const { navigate } = this.props.navigation
+    if (this.state.codex       === '' ||
+        this.state.temperature === '' ||
+        this.state.course      === ''
+       ) {
+      Alert.alert('Error: please enter a value for all text fields.')
+    }
+    else {
+      navigate('LineupForm', { title: this.state.position, props: this.state })
+    }
+  }
+
+  render() {
+    return (
+      <View style={styles.container}>
+        <View style={styles.row}>
+          <Text style={styles.text}>
+            Race Codex:
+          </Text>
+          <TextInput
+            style={styles.input}
+            value={this.state.codex}
+            placeholder={'N/a'}
```

Comment:
>'N/a' [](start = 25, length = 5)

'N/A' is not a great placeholder - typically placeholders inform the user what needs to be entered. If it's optional, you can signal with other context surrounding the TextInput (e.g., req'd fields marked as such)

Reply:
Fix for other TextInput as well.

---
In reply to: [157104889](https://github.com/rozele/React-Native-Ski-Team/pull/2#discussion_r157104889) [](ancestors = 157104889)